frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -1,4 +1,106 @@
 {
+  "Disable": "Disable",
+  "Enable": "Enable",
+  "CatalogSource details": "CatalogSource details",
+  "Publisher": "Publisher",
+  "Availability": "Availability",
+  "Denotes whether this CatalogSource provides operators to a specific namespace, or the entire cluster.": "Denotes whether this CatalogSource provides operators to a specific namespace, or the entire cluster.",
+  "The ConfigMap, image, or address for this CatalogSource's registry.": "The ConfigMap, image, or address for this CatalogSource's registry.",
+  "Registry poll interval": "Registry poll interval",
+  "Number of Operators": "Number of Operators",
+  "The number of packages this CatalogSource provides.": "The number of packages this CatalogSource provides.",
+  "Operators": "Operators",
+  "Package not found": "Package not found",
+  "Cannot create a Subscription to a non-existent package.": "Cannot create a Subscription to a non-existent package.",
+  "Endpoint": "Endpoint",
+  "# of Operators": "# of Operators",
+  "Disabled": "Disabled",
+  "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.": "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.",
+  "Enable source": "Enable source",
+  "Cluster wide": "Cluster wide",
+  "Create {{item}}": "Create {{item}}",
+  "Edit Subscription": "Edit Subscription",
+  "Uninstall Operator": "Uninstall Operator",
+  "Cannot update": "Cannot update",
+  "CatalogSource was removed.": "CatalogSource was removed.",
+  "Unknown": "Unknown",
+  "All Namespaces": "All Namespaces",
+  "Managed Namespaces": "Managed Namespaces",
+  "{{count}} Namespaces": "{{count}} Namespaces",
+  "{{count}} Namespaces_plural": "{{count}} Namespaces",
+  "None": "None",
+  "No Operators found": "No Operators found",
+  "No Operators are available{activeNamespace !== ALL_NAMESPACES_KEY && (\n            <>\n              &nbsp;for project <strong>{activeNamespace}</strong>\n            </>\n          )}.": "No Operators are available{activeNamespace !== ALL_NAMESPACES_KEY && (\n            <>\n              &nbsp;for project <strong>{activeNamespace}</strong>\n            </>\n          )}.",
```

Comment:
I would avoid embedding logic into the message itself using `Trans`. Probably we want two separate complete messages, one when all namespaces is selected and another when a specific project is selected.